doc/spec/manifest.md

```diff
@@ -0,0 +1,249 @@
+
+        - **`key`** *string*
+
+            The hex encoding of the key required to decrypt the content.
+
+        - **`iv`** *string*
+
+            The hex encoding of the initialization vector required to decrypt
+            the content. If omitted and an initialization vector is required
+            for the specified algorithm, assume an all-zero initialization
+            vector; this may be used to support convergent encryption.
+    
+    - **`digests`** *object*
+    
+        An object mapping supported digest algorithms to the hex encoded
+        value of that digest for the content. If the content is encrypted,
+        then the digest is of the encrypted content.
+    
+    Any media type may be in the `mediaType` field as the application may
+    use any type of content. If the `mediaType` is
+    `application/vnd.docker.distribution.content.manifest.v1+json` then the
+    dependency is another content manifest as described by this document. The
+    dependencies from that manifest should also be considered dependencies of
+    this application.
+    
+    If the `repository` field is empty or omitted, then the object should be
+    considered to be in the same repository as this manifest.
+
+    Encryption combined with a verifiable digest of the content should allow
+    for both confidentiality and authentication of data.
+
+- **`application`** *object*
+
+    Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the JSON object in the `data` field.
+    
+    - **`data`** *object*
+    
+        Abstract application data in any schema, as described by the
+        `mediaType` field.
+    
+    Application data may be *any* JSON object, allowing this manifest format,
+    along with the generic dependencies list, to support any type of 
+    application that can be represented as a JSON configuration and a
+    collection of content-addressable blobs of data.
+
+- **`metadata`** *object*
+
+    Metadata may be *any* JSON object, allowing content creators to annotate
+    their application manifest with any additional labels beyond those already
+    defined by this manifest format. 
+
+
+### Example Manifest
+
+```json
+{
+    "schemaVersion": 2,
+    "createdAt": "2015-02-11T15:07:17.639302443-08:00",
+    "maintainers": [
+        {
+            "name": "Phillip Edward Nuñez",
+            "email": "pnunez@example.com",
+            "url": "https://example.com/pnunez/"
+        }
+    ],
+    "description": "Application to Simulate Turbo Encabulators",
+    "keywords": [
+        "foo",
+        "bar",
+        "baz"
+    ],
+    "website": "https://foo.bar.baz.example.com",
+    "licenses": [
+        "Apache-2.0"
+    ],
+    "dependencies": [
+        {
+            "mediaType": "application/vnd.docker.distribution.content.manifest.v1+json",
+            "size": 1024,
+            "repository": "isv.example.com/base/system",
+            "digests": {
```

Comment:
Can these just be a list of strings, rather than requiring another level indirection?

```
"digests": [
    "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7",
    "tarsum.v1+sha256:..."
]
```